bubuku/id_extractor.py

```diff
@@ -24,17 +32,11 @@ def get_broker_id(self):
         while not os.path.isfile(meta_path):
             return None
         with open(meta_path) as f:
-            lines = f.readlines()
-            for line in lines:
-                match = re.search('broker\.id=(\d+)', line)
-                if match:
-                    self.broker_id = match.group(1)
-                    return self.broker_id
-        return None
+            self.broker_id = _search_broker_id(f.readlines())
+        return self.broker_id
```

Comment:
will it return None as before in case there is no match ?

Reply:
i see tests 👍

Reply:
Yes, as it did before. I was trying to understand the logic behind it. Failed and renamed the class.